package.json

```
@@ -13,14 +13,14 @@
         "images": "imagemin '_site/assets/images' --out-dir='_site/assets/images'",
         "watch:sass": "sass --watch src/assets/scss:src/assets/css",
         "watch:eleventy": "eleventy --serve --port=2022",
-        "build:sass": "sass src/assets/scss:src/assets/css",
+        "build:sass": "sass --no-source-map --style=compressed src/assets/scss:src/assets/css",
         "build:eleventy": "npx @11ty/eleventy",
         "fetch:team": "node tools/fetch-team-data.js",
         "fetch:stats": "node tools/fetch-stats.js",
         "fetch:sponsors": "node tools/fetch-sponsors.js",
         "fetch": "npm-run-all --parallel fetch:*",
-        "start": "npm-run-all build:sass --parallel watch:*  --parallel images",
-        "build": "npm-run-all build:sass --parallel build:*   --parallel images"
+        "start": "npm-run-all build:sass --parallel watch:* images",
```

If we run `images` in parallel with eleventy, it will probably do nothing because it operates on a directory that eleventy has yet to copy from src.

I'm not sure if it was doing anything in the previous version as well. It looks like it wasn't run at all, and even if it is then it's after we finished watching. Maybe we should just remove it?

---

"start": "npm-run-all build:sass images --parallel watch:*",

May be we should do something like this. Run the sass build and then images build and start watching in parallel?

---

`images` works on `_site` which is initially empty so running `images` at that point would do nothing?

---

I was thinking if a new image was added during the watch mode the image command would be run only once? Should we may be consider removing that in the watch mode since it is run only once? or do it once the parallel commands execution was completed? like this
```suggestion
        "start": "npm-run-all build:sass --parallel watch:* --serial images",
```

---

Then the minified images won't be used in the local preview, and Eleventy will overwrite them on subsequent `npm start`. I think we just can't incorporate `images` in the watch mode, because it can be run either too early or too late, and it doesn't have its own watch mode. In my opinion, we should remove `images` from this script because it has no effect.

---

Done!